lib/helper/Playwright.js

```diff
@@ -0,0 +1,2423 @@
+    assertElementExists(els, context);
+    matcher = els[0];
+  }
+  const els = await findClickable.call(this, matcher, locator);
+  if (context) {
+    assertElementExists(els, locator, 'Clickable element', `was not found inside element ${new Locator(context).toString()}`);
+  } else {
+    assertElementExists(els, locator, 'Clickable element');
+  }
+  await els[0].click(options);
+  const promises = [];
+  if (options.waitForNavigation) {
+    promises.push(this.waitForNavigation());
+  }
+  promises.push(this._waitForAction());
+  return Promise.all(promises);
+}
+
+async function findClickable(matcher, locator) {
+  if (locator.react) return findReact(matcher, locator);
+  locator = new Locator(locator);
+  if (!locator.isFuzzy()) return findElements.call(this, matcher, locator);
+
+  let els;
+  const literal = xpathLocator.literal(locator.value);
+
+  els = await findElements.call(this, matcher, Locator.clickable.narrow(literal));
+  if (els.length) return els;
+
+  els = await findElements.call(this, matcher, Locator.clickable.wide(literal));
+  if (els.length) return els;
+
+  try {
+    els = await findElements.call(this, matcher, Locator.clickable.self(literal));
+    if (els.length) return els;
+  } catch (err) {
+    // Do nothing
+  }
+
+  return findElements.call(this, matcher, locator.value); // by css or xpath
+}
+
+async function proceedSee(assertType, text, context, strict = false) {
+  let description;
+  let allText;
+  if (!context) {
+    let el = await this.context;
+
+    if (el && !el.getProperty) {
+      // Fallback to body
+      el = await this.context.$('body');
+    }
+
+    allText = [await el.getProperty('innerText').then(p => p.jsonValue())];
+    description = 'web application';
+  } else {
+    const locator = new Locator(context, 'css');
+    description = `element ${locator.toString()}`;
+    const els = await this._locate(locator);
+    assertElementExists(els, locator.toString());
+    allText = await Promise.all(els.map(el => el.getProperty('innerText').then(p => p.jsonValue())));
+  }
+
+  if (strict) {
+    return allText.map(elText => equals(description)[assertType](text, elText));
+  }
+  return stringIncludes(description)[assertType](text, allText.join(' | '));
+}
+
+async function findCheckable(locator, context) {
+  let contextEl = await this.context;
+  if (typeof context === 'string') {
+    contextEl = await findElements.call(this, contextEl, (new Locator(context, 'css')).simplify());
+    contextEl = contextEl[0];
+  }
+
+  const matchedLocator = new Locator(locator);
+  if (!matchedLocator.isFuzzy()) {
+    return findElements.call(this, contextEl, matchedLocator.simplify());
+  }
+
+  const literal = xpathLocator.literal(locator);
+  let els = await findElements.call(this, contextEl, Locator.checkable.byText(literal));
+  if (els.length) {
+    return els;
+  }
+  els = await findElements.call(this, contextEl, Locator.checkable.byName(literal));
+  if (els
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['bc' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4904555)